### PR TITLE
Misc fixes for video player

### LINKED ISSUE #8677: 3.17.0 RC1: Video doesn't stop when minimizing the app
Oneplus 7T und 5T
It playes in Background, I can head the sound until I kill the nextcloud Appp

### LINKED ISSUE #9451: Video audio continues playing if app switcher is used while video is playing
### Steps to reproduce
1. Start playing a video
2. While the video is still playing, use the Android app switcher to switch to another app

### Expected behaviour
- Video playback should pause when the app switcher is invoked
- The app should remember the place in the video at the time it was paused when switching back and forth between apps

### Actual behaviour
- The video audio continues playing in the background until the video ends or the app is force closed
- If you try to switch back to the Nextcloud app, the video player acts as though you have just opened the file. If you tap play, multiple copies of the audio will play over each other
- If you pause a video before switching to another  app, and then switch back to the Nextcloud app, the the video player starts the video at the beginning

### Environment data
Android version: 11

Device model: Samsung Galaxy S20

Stock or customized system: Stock

Nextcloud app version: 3.18.0

### LINKED ISSUE #9793: Draft Video Full Screen Continues to Play and Crashes with OutOfMemory
### Steps to reproduce
1. Start playing video
2. Open it full screen
3. Exit using a gesture to the left

### Expected behaviour
- I shouldn't hear audio from the video, it should stop.
- I shouldn't see OutOfMemoryError

### Actual behaviour
- I hear the audio from the video, then I see OutOfMemory error

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

### PR DESCRIPTION
**Pull request by AlvaroBrey**
Fixes #9793
Fixes #9451
Fixes #8677

- [x] Tests written, or not not needed

### CODE REVIEW

File: src/main/res/layout/exo_player_control_view.xml [hunk 1]
```diff
@@ -31,38 +31,22 @@
...
             android:id="@id/exo_play"
+            android:foreground="?attr/selectableItemBackgroundBorderless"
             style="@style/ExoMediaButton.Play" />
 
         <ImageButton
             android:id="@id/exo_pause"
+            android:foreground="?attr/selectableItemBackgroundBorderless"
             style="@style/ExoMediaButton.Pause" />
 
-        <ImageButton
```
Comment (tobiasKaminsky) [inline src/main/res/layout/exo_player_control_view.xml:56]: This button works in preview fragment (not fullscreen).

Comment (AlvaroBrey) [reply src/main/res/layout/exo_player_control_view.xml:56]: Preview fragment does not use this custom control view, but rather the default control view from ExoPlayer (with an added fullscreen toggle).

If we want to keep feature parity between fullscreen and fragment, we need to either:
a) Implement these buttons manually for fullscreen, or
b) Use the default controls for fullscreen as we're doing in the fragment.

**Review by tobiasKaminsky — APPROVED** [review]

**Comment by jakobroehrl** [comment]
@AlvaroBrey 
This is working very nice!

One last thing: If I play a long video and display timeout starts the video is stopped. 
Maybe it would nice to keep the display on while the video is playing. 
I can add a new issue if you want.

**Comment by tobiasKaminsky** [comment]
> I can add a new issue if you want.

Please do this. And thanks for testing

**Comment by AlvaroBrey** [comment]
> @AlvaroBrey This is working very nice!
> 
> One last thing: If I play a long video and display timeout starts the video is stopped. Maybe it would nice to keep the display on while the video is playing. I can add a new issue if you want.

Already tracking that in #9059

**Pull request merged**